package.json

```diff
@@ -1,37 +1,42 @@
 {
-    "name": "ace",
-    "description": "Ajax.org Code Editor is a full featured source code highlighting editor that powers the Cloud9 IDE",
-    "version": "1.1.9",
-    "homepage": "http://github.com/ajaxorg/ace",
-    "engines": {"node": ">= 0.6.0"},
-    "author": "Fabian Jakobs <fabian@c9.io>",
-    "main": "lib/ace",
-    "repository": {
-        "type": "git",
-         "url": "http://github.com/ajaxorg/ace.git"
-    },
-    "dependencies": {
-        "mime": "1.2.x"
-    },
-    "devDependencies": {
-        "asyncjs": "0.0.x",
-        "jsdom": "0.2.x",
-        "amd-loader": "~0.0.4",
-        "dryice": "0.4.11",
-        "architect-build": "https://github.com/c9/architect-build/tarball/a3bad51808"
-    },
-    "mappings": {
-        "ace": "."
-    },
-    "licenses": [{
-        "type": "BSD New",
-        "url": "http://opensource.org/licenses/BSD-3-Clause"
-    }],
-    "directories": {
-        "lib": "lib/ace"
-    },
-    "scripts": {
-        "start": "node static.js",
-        "test": "node lib/ace/test/all.js"
+  "name": "ace",
+  "description": "Ajax.org Code Editor is a full featured source code highlighting editor that powers the Cloud9 IDE",
+  "version": "1.1.9",
+  "homepage": "http://github.com/ajaxorg/ace",
+  "engines": {
+    "node": ">= 0.6.0"
+  },
+  "author": "Fabian Jakobs <fabian@c9.io>",
+  "main": "lib/ace",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/ajaxorg/ace.git"
+  },
+  "dependencies": {
+    "mime": "1.2.x"
+  },
+  "devDependencies": {
+    "amd-loader": "~0.0.4",
+    "architect-build": "https://github.com/c9/architect-build/tarball/a3bad51808",
+    "asyncjs": "0.0.x",
+    "concat": "^1.0.0",
```

Comment:
Current system is hard programmed in a Makefile for generating output files.

I myself wanted to just pickup 5-10 modes and 3-4 workers + the ace core and some handy extensions and push them through concat and then mangle the code. That way it's all delivered efficiently through the browser.

Grunt file would help clarify such a process.